.jenkins/Jenkinsfile

```diff
@@ -375,71 +377,89 @@ properties([buildDiscarder(logRotator(artifactDaysToKeepStr: '90',
+             "ACC1804 clang-7 Release LVI" :                            { ACCTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'clang-7', 'Release', 'ControlFlow') },
+             "ACC1804 gcc Debug" :                                      { ACCTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'gcc', 'Debug') },
+             "ACC1804 gcc Release" :                                    { ACCTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'gcc', 'Release') },
+             "ACC1804 gcc Debug LVI" :                                  { ACCTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'gcc', 'Debug', 'ControlFlow') },
+             "ACC1804 gcc Release LVI" :                                { ACCTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'gcc', 'Release', 'ControlFlow') },
+             "ACC1804 Container RelWithDebInfo" :                       { ACCContainerTest(AGENTS_LABELS["acc-ubuntu-18.04"], '18.04') },
+             "ACC1804 Container RelWithDebInfo LVI" :                   { ACCContainerTest(AGENTS_LABELS["acc-ubuntu-18.04"], '18.04', 'ControlFlow') },
+             "ACC1804 Package RelWithDebInfo" :                         { ACCPackageTest(AGENTS_LABELS["acc-ubuntu-18.04"], '18.04') },
+             "ACC1804 Package RelWithDebInfo LVI" :                     { ACCPackageTest(AGENTS_LABELS["acc-ubuntu-18.04"], '18.04', 'ControlFlow') },
+             "ACC1804 GNU gcc SGX1FLC" :                                { ACCGNUTest() },
+             "AArch64 1604 GNU gcc Debug" :                             { AArch64GNUTest('16.04', 'Debug')},
+             "AArch64 1604 GNU gcc Release" :                           { AArch64GNUTest('16.04', 'Release')},
+             "AArch64 1804 GNU gcc Debug" :                             { AArch64GNUTest('18.04', 'Debug')},
+             "AArch64 1804 GNU gcc Release" :                           { AArch64GNUTest('18.04', 'Release')},
+             "Sim 1804 clang-7 SGX1 Debug" :                            { simulationTest('18.04', 'SGX1', 'Debug')},
+             "Sim 1804 clang-7 SGX1 Release" :                          { simulationTest('18.04', 'SGX1', 'Release')},
+             "Sim 1804 clang-7 SGX1-FLC Debug" :                        { simulationTest('18.04', 'SGX1FLC', 'Debug')},
+             "Sim 1804 clang-7 SGX1-FLC Release" :                      { simulationTest('18.04', 'SGX1FLC', 'Release')},
+             // "RHEL-8 simulation clang-8 SGX1 Release":             { RHEL8Test('clang', 'Release', ["OE_SIMULATION=1"]) }, // Enable when https://github.com/openenclave/openenclave/issues/2556 is fixed.
+             "RHEL-8 simulation clang-8 SGX1 Debug":                    { RHEL8Test('clang', 'Debug',   ["OE_SIMULATION=1"]) },
+             // "RHEL-8 simulation gcc-8 Release":                    { RHEL8Test('gcc',   'Release', ["OE_SIMULATION=1"]) }, // Enable when https://github.com/openenclave/openenclave/issues/2558 is fixed.
```

Comment:
The comment says to uncomment this line and line 461 after #2558 is fixed, and that issue is marked as closed.

Reply:
Yep, you're right!

This is not part of the work with this PR, but I think @oprinmarius can uncomment the stages with no issues at this point.

Reply:
There is another PR open for this

Reply:
@dthaler 

This is the PR https://github.com/openenclave/openenclave/pull/2754.

@BRMcLaren, I notice you have it into `Draft` state. `Bors try` worked on it, so I think we can merge it.